redis/types.go

```diff
@@ -0,0 +1,37 @@
+package redis
+
+import (
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+// Config for redis connection
+// TODO Set all default values
+type Config struct {
+	Host string `yaml:"host" env:"REDIS_HOST" env-description:"Redis host"`
+	Port string `yaml:"port" env:"REDIS_PORT" env-description:"Redis port"`
+	Pass string `env:"REDISPASS" env-description:"Redis user password"`
+}
+
+// repository represent the repository model
+type repository struct {
+	Client redis.Cmdable
+}
+
+// Repository represent the repositories
+type Repository interface {
```

Comment:
Let's rename this interface KVStore and move it out of redis package, let's create a "type" package to avoid cyclic dependencies.

Reply:
OK. I've changed `Repository` to be `KVStore` but I'm not moving out from redis pkg yet.